posts/tests.py

```diff
@@ -96,44 +70,41 @@ def test_search(self):
         self.assertFalse(posts.exists())
         self.assertEqual(posts.count(), 0)
 
+class PostEditTestCase(TestCase):
+    def setUp(self):
+        self.client = Client()
```

Comment:
@sehmuzc These tests are failing because imports are incomplete. Please make sure that they work next time. Also linting is failing too.